homeassistant/components/blebox/climate.py

```diff
@@ -40,23 +53,31 @@ class BleBoxClimateEntity(BleBoxEntity[blebox_uniapi.climate.Climate], ClimateEn
     """Representation of a BleBox climate feature (saunaBox)."""
 
     _attr_supported_features = ClimateEntityFeature.TARGET_TEMPERATURE
-    _attr_hvac_modes = [HVACMode.OFF, HVACMode.HEAT]
     _attr_temperature_unit = TEMP_CELSIUS
 
+    @property
+    def hvac_modes(self):
+        """Return list of supported HVAC modes."""
+        return [HVACMode.OFF, self.hvac_mode]
```

Comment:
Should this not return the list of all modes that can be selected rather than just either current mode & off? So something like:
```suggestion
        """Return list of supported HVAC modes."""
        return BLEBOX_TO_HVACMODE.values()
```

Reply:
Unfortunately no, we think that changing device mode shouldn't be allowed through HA. 
Also to keep consistency with previous integrations, the method `async_set_hvac_mode` is used to make the device operate(eg. run heating), not to change the mode.